src/js/constants.js

```diff
@@ -1,5 +1,5 @@
-//export const STUY_SPEC_API_URL = 'https://api.stuyspec.xyz';
-export const STUY_SPEC_API_URL = 'http://localhost:3000';
+export const STUY_SPEC_API_URL = 'https://api.stuyspec.xyz';
+//export const STUY_SPEC_API_URL = 'http://localhost:3000';
```

Comment:
Use your localhost. After you finish LoginModalOverlay stuff, figure out how to run the stuy-spec-api on your machine.